webapi/Plugins/Chat/ApiConnectorPlugin.cs

```diff
@@ -0,0 +1,166 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using CopilotChat.WebApi.Options;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel;
+
+namespace CopilotChat.WebApi.Plugins.Chat;
+
+/// <summary>
+/// This class is a plugin that connects to an API.
+/// </summary>
+public sealed class ApiConnectorPlugin
```

Comment:
Can we rename this to `GraphOboPlugin`? `ApiConnectorPlugin` is too ambiguous as most plugins hook up some type of API. Then change all instances and comments accordingly

Reply:
done - renamed to MsGraphOboPlugin to match the  other plugin named MsGraph